server/datastore/mysql/vulnerabilities.go

```diff
@@ -309,11 +309,11 @@ func (ds *Datastore) ListVulnerabilities(ctx context.Context, opt fleet.VulnList
 func (ds *Datastore) CountVulnerabilities(ctx context.Context, opt fleet.VulnListOptions) (uint, error) {
 	selectStmt := `
 		SELECT
-			COUNT(*)
+			COUNT(DISTINCT combined.cve)
 		FROM (
-			SELECT cve, created_at, source FROM software_cve
+			SELECT cve FROM software_cve
 			UNION
-			SELECT cve, created_at, source FROM operating_system_vulnerabilities
+			SELECT cve FROM operating_system_vulnerabilities
```

Comment:
Makes sense, count(*) will count all the rows from the UNION. 
Why remove the created_at/source columns? I don't know if there are any rows that fall in this criteria, but technically UNION will combine rows if the same combination of `(cve, created_at, source)` exist right?

Reply:
Given that there's a subquery in there already, thinking that we're close enough to optimized for these purposes (based on benchmarking I did locally), so my current thought is to leave things as-is vs. tweaking/QA'ing again.